tasks/cycles/cycles.js

```diff
@@ -6,30 +6,78 @@
  * - 351  - 1350: 15%
  * - 1351 - 2700: 30%
  * - 2701 - 6500: 45%
+ * - 6501 - 9999: 60% // added additional range to cover 0 - 9999 values range
  * @param {number} redemption amount (0 - 9999)
  * @returns {number} discount in percent
  */
 
 function calculateDiscount(redemption) {
   let discount;
-
-  //PLACE YOUR CODE HERE
-
-  return discount;
+  if (typeof redemption === "number") {
```

Comment:
it's okay but hard to read, it's better to write several if..else statements than trying to wrap it in the construction.
if (typeof redemption === "number") {
 redemption = (Math.floor(redemption));
} else {
 discount = 0;
}

if (redemption >= 6501 & redemption <= 9999) {...}